sdk/python/endpoints/online/managed/online-endpoints-inference-schema.ipynb

```diff
@@ -0,0 +1,753 @@
+    "\n",
+    "Each of the deployments in this example use the same model and produce the same output, however they each accept input in different formats.\n",
+    "\n",
+    "For the first deployment, we will use the [score-numpy.py](../../../../../cli/endpoints/online/managed/inference-schema/code/score-numpy.py) scoring script, which declares a single input parameter `iris` as a `NumpyParameterType` and a nested `StandardParameterType` as output.\n",
+    "\n",
+    "```python\n",
+    "@input_schema(\n",
+    "    param_name=\"iris\",\n",
+    "    param_type=NumpyParameterType(np.array([[7.2, 3.2, 6.0, 1.8]]))\n",
+    ")\n",
+    "@output_schema(\n",
+    "    output_type=StandardPythonParameterType({\n",
+    "        \"Category\" : [\"Virginica\"]\n",
+    "    })\n",
+    ")\n",
+    "``` \n",
+    "When the following [sample input](../../../../../cli/endpoints/online/managed/inference-schema/sample-inputs/numpy.json) is sent, it is automatically validated and converted to a Numpy array. Validation can be toggled using the `enforce_column_type` and `enforce_shape` arguments to `NumpyParameterType`.\n",
+    "\n",
+    "```json\n",
+    "{\"iris\": [[7.2, 3.2, 6.0, 1.8], [4.2, 3.5, 1.0, 3.0]]}\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.1 Define the deployment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deployment = ManagedOnlineDeployment(\n",
+    "    name=\"infsrv-numpy\",\n",
+    "    endpoint_name=endpoint_name,\n",
+    "    model=f\"azureml:{model.name}:{model.version}\",\n",
+    "    code_configuration=CodeConfiguration(\n",
+    "        code=code_path, scoring_script=\"score-numpy.py\"\n",
+    "    ),\n",
+    "    environment=Environment(\n",
+    "        image=\"mcr.microsoft.com/azureml/minimal-ubuntu20.04-py38-cpu-inference\",\n",
+    "        conda_file=conda_file_path,\n",
+    "    ),\n",
+    "    instance_type=\"Standard_DS2_v2\",\n",
+    "    instance_count=1\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.2 Create the deployment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "poller = ml_client.online_deployments.begin_create_or_update(deployment)\n",
+    "poller.wait()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.3 Confirm that creation was successful"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "status = poller.status()\n",
+    "if status != \"Succeeded\":\n",
+    "    raise DeploymentException(status)\n",
+    "else:\n",
+    "    print(\"Deployment creation succeeded\")\n",
+    "    deployment = poller.result()\n",
+    "    print(deployment)"
+   ]
+  },
```

Comment:
Do we have any specific reasons for using `wait()` and checking the status instead of just using `ml_client.online_deployments.begin_create_or_update(deployment).result()` as well as other examples?

Reply:
This is aligned now - thank you!